app/src/main/kotlin/com/apkupdater/data/github/GitHubApp.kt

```diff
@@ -45,6 +45,7 @@ val GitHubApps = listOf(
     GitHubApp("com.apkupdater.ci", "rumboalla", "apkupdater"),
     GitHubApp("com.standardnotes", "standardnotes", "app"),
     GitHubApp("com.xBrowserSync.android", "xbrowsersync", "app"),
+	GitHubApp("io.appflowy.appflowy", "AppFlowy-IO", "AppFlowy"),
```

Comment:
Not properly tabbed

Reply:
Normally it's fixed, i also added one more app